app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/conversationList.tsx

```
@@ -27,7 +27,7 @@ type ListItem = ConversationItem & { isNew?: boolean };
 export const List = () => {
   const [conversationSlug] = useQueryState("id");
   const { searchParams, input } = useConversationsListInput();
-  const { conversationListData, navigateToConversation, isPending, isFetchingNextPage, hasNextPage, fetchNextPage } =
+  const { conversationListData, navigateToConversation, isPending, isFetching, isFetchingNextPage, hasNextPage, fetchNextPage } =
     useConversationListContext();
```

ESLint

[error] Replace `·conversationListData,·navigateToConversation,·isPending,·isFetching,·isFetchingNextPage,·hasNextPage,·fetchNextPage·}·=⏎···` with `⏎····conversationListData,⏎····navigateToConversation,⏎····isPending,⏎····isFetching,⏎····isFetchingNextPage,⏎····hasNextPage,⏎····fetchNextPage,⏎··}·=`

(prettier/prettier)